curator/app.js

```diff
@@ -24,8 +23,10 @@ module.exports = function (db) {
   /*Serving static content*/
   app.use(express.static(__dirname + '/views/static'))
 
```

Comment:
This is probably the reason why you don't see images. this line of code tells express to look for static content in the views/static directory of the app. Don't worry about this though cause we are gonna handle static files with nginx pretty soon